vms/shift/utils.py

```diff
@@ -75,6 +75,27 @@ def create_volunteer_with_details(volunteer):
     return v1
 
 
+def create_volunteer_with_details_dynamic_password(volunteer):
+    """
```

Comment:
please no. What is the use case of passing unrelated data in a list ? Please use a proper dict :-(

Reply:
I was thinking for the same, (I hate using lists).
but the above related functions were using list so I didn't want to change the similarities.

Reply:
Also @tonythomas01 can you please help me solve those broken pipe issue in travis without increasing the build time (it already 7 mins and I've added only 1 app).

Reply:
okey - in that case, can you create a different task for changing this abuse of lists to dicts ? Let me look at the build steps as well.

Reply:
Okay I looked on the dependency of this change. Its quite huge should we really change it?

Reply:
I am not sure about other instances, but if a function receives data about an instance and populates the instance model to create a new one - that is a dictionary use case. It would be huge depending on how much things are here, but hopefully somebody else would take care of it I guess.

Reply:
Sure then I'll create an issue for it. :+1: .
Also should the form filling in e2e tests is a dict use case right? (I mean when we pass info to that fill function) We use `send_keys()` to send the info at the end.

Reply:
I think yes. See, when its structured data - yes dict. Collection of similar stuff - list. Even though, there is nothing wrong in having mixed data in a list though, like say `awkward_list = [1, 2, 'myvar', {'a_json_key': 'a_json_value'}`, 345.4]`

Reply:
Thanks. that makes things quite clear.